src/ReportPortal.Shared/Reporter/Http/HttpClientHandlerFactory.cs

```diff
@@ -36,10 +36,18 @@ public virtual HttpClientHandler Create()
 
             httpClientHandler.Proxy = GetProxy();
 
+            var ignoreSslErrors = Configuration.GetValue<bool>("Server:IgnoreSslErrors", true);
```

Comment:
I even don't mind to make it `false` by default :) Not sure we should break users... how many users we may break. In any case let's:
- Keep `true` by default
- Add mention about new configuration property in `docs` folder

Reply:
I'd prefer to set it to `false`, but I was being cautious.
Depends on how much of a breaking change the team is willing to accept :)

Reply:
This one we can easily accept. It's a matter of how we notify users. Will do via release notes. So, let's use recommended way by default?

Reply:
I can get behind that 👍. Done.